src/wasm/tests/BUILD

```diff
@@ -1,12 +1,18 @@
 load("//third_party/java/arcs/build_defs:build_defs.bzl", "arcs_ts_test")
 
+exports_files(["manifest.arcs"])
+
+filegroup(
+    name = "manifest_srcs",
+    srcs = ["manifest.arcs"],
+)
```

Comment:
I don't think you need this filegroup at all anymore. AFAIK, you should be able to just add "manifest.arcs" as a dep below.

Reply:
Tried that; didn't work:
```
ERROR: /usr/local/google/home/mykmartin/code/arcs/src/wasm/tests/BUILD:10:1: in deps attribute of run_in_repo_test rule //src/wasm/tests:wasm-api-test: source file '//src/wasm/tests:manifest.arcs' is misplaced here (expected no files).
```

I also tried adding it as a `data` field, no dice.